src/sveta/Test_2_StringEvenNo.java

```diff
@@ -0,0 +1,21 @@
+package sveta;
+
+public class Test_2_StringEvenNo {
+
+	public static void evenNo(String str) {
+		char ch;
+		int input=0;
+		for(int i=0;i<=str.length()-1;i++) {
```

Comment:
Here, you want last even number, so best approach is to reverse the loop

Reply:
ok will do it.